packages/shared/lib/Cargo.toml

```diff
@@ -18,7 +18,7 @@ nodejs = []
 web = []
 
 [build-dependencies]
-namada_tx = { git = "https://github.com/anoma/namada", tag = "v0.46.0" }
+namada_tx = { git = "https://github.com/anoma/namada", rev = "49a4a5d3260423df19ead14df82d18a51fa9b157" }
```

Comment:
Can this be pointed to `v0.46.1`, or are we waiting on the next release?

Reply:
I think this one is later probably like 1.0.1 or sth :)

Reply:
But it should be safe to merge I think.